docs/dataout.rst

```diff
@@ -0,0 +1,159 @@
+| S3_REGION     | us-west-1                    | S3 region            |
++---------------+------------------------------+----------------------+
+| S3_BUCKET     | lega                         | S3 bucket to use     |
++---------------+------------------------------+----------------------+
+| S3_SECURE     | true                         | true if S3 backend   |
+|               |                              | should be accessed   |
+|               |                              | over HTTPS           |
++---------------+------------------------------+----------------------+
+| S3_R          | /etc/s                       | Path to the CA certs |
+| OOT_CERT_PATH | sl/certs/ca-certificates.crt | file for S3          |
+|               |                              | connectivity         |
++---------------+------------------------------+----------------------+
+| ARCHIVE_PATH  | /                            | Path to the          |
+|               |                              | filesystem-archive   |
++---------------+------------------------------+----------------------+
+| PASSPORT_PU   | /etc/ega/jwt/passport.pem    | Path to the public   |
+| BLIC_KEY_PATH |                              | key for passport JWT |
+|               |                              | validation           |
++---------------+------------------------------+----------------------+
+| OPENID_CONF   |                              | URL of the OpenID    |
+| IGURATION_URL |                              | configuration        |
+|               |                              | endpoint             |
++---------------+------------------------------+----------------------+
+| VISA_PU       | /etc/ega/jwt/visa.pem        | Path to the public   |
+| BLIC_KEY_PATH |                              | key for visas JWT    |
+|               |                              | validation           |
++---------------+------------------------------+----------------------+
+| CRYPT4GH_PRI  | /etc/ega/crypt4gh/key.pem    | Path to the Crypt4GH |
+| VATE_KEY_PATH |                              | private key          |
++---------------+------------------------------+----------------------+
+| CRYPT4GH      | /etc/ega/crypt4gh/key.pass   | Path to the Crypt4GH |
+| _PRIVATE_KEY_ |                              | private key          |
+| PASSWORD_PATH |                              | passphrase           |
++---------------+------------------------------+----------------------+
+| LOGSTASH_HOST |                              | Hostname of the      |
+|               |                              | Logstash instance    |
+|               |                              | (if any)             |
++---------------+------------------------------+----------------------+
+| LOGSTASH_PORT |                              | Port of the Logstash |
+|               |                              | instance (if any)    |
++---------------+------------------------------+----------------------+
```

Comment:
All of these are rendered as one line, The brakes in the ENVs is rendered as spaces which makes the text look odd.

Reply:
solved